testnets/pocket/chain.json

```diff
@@ -0,0 +1,116 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "pocket-beta",
+  "status": "live",
+  "network_type": "testnet",
+  "website": "https://www.pokt.network/",
+  "pretty_name": "Pocket Beta TestNet",
+  "chain_type": "cosmos",
+  "chain_id": "pocket-beta",
+  "bech32_prefix": "pokt@",
+  "daemon_name": "poktrolld",
+  "node_home": "$HOME/.poktroll",
+  "key_algos": ["secp256k1"],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "upokt",
+        "fixed_min_gas_price": 1
```

Comment:
@Olshansk let's do
```
        "fixed_min_gas_price": 0.001,
        "low_gas_price": 0.001,
        "average_gas_price": 0.002,
        "high_gas_price": 0.0025
```
Reasoning:
- Current real config value is `0.000000001upokt`
- `0.001` as a minimum fee is high enough to let most transactions in, just in case. This is a still a small fee, though.

Reply:
This is great, ty!